src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$dayGap};
+  width: getWidth($daySize, $dayGap);
+  margin: 10px auto;
```

Comment:
[BEM & STYLES] - Don't add external styles (positioning or `margins`) to BEM-blocks. Use mix where necessary and move all external styles under element selector.